pkg/controllers/scheduler/profile.go

```diff
@@ -30,19 +32,94 @@ import (
 	"github.com/kubewharf/kubeadmiral/pkg/controllers/scheduler/framework/runtime"
 )
 
-func (s *Scheduler) profileForFedObject(_ *unstructured.Unstructured, handle framework.Handle) (framework.Framework, error) {
-	// TODO: generate registry from fed object's type
-	DefaultRegistry := runtime.Registry{
-		apiresources.APIResourcesName:                           apiresources.NewAPIResources,
-		tainttoleration.TaintTolerationName:                     tainttoleration.NewTaintToleration,
-		clusterresources.ClusterResourcesBalancedAllocationName: clusterresources.NewClusterResourcesBalancedAllocation,
-		clusterresources.ClusterResourcesFitName:                clusterresources.NewClusterResourcesFit,
-		clusterresources.ClusterResourcesLeastAllocatedName:     clusterresources.NewClusterResourcesLeastAllocated,
-		rsp.ClusterCapacityWeightName:                           rsp.NewClusterCapacityWeight,
-		placement.PlacementFilterName:                           placement.NewPlacementFilter,
-		clusteraffinity.ClusterAffinityName:                     clusteraffinity.NewClusterAffinity,
-		maxcluster.MaxClusterName:                               maxcluster.NewMaxCluster,
-	}
-
-	return runtime.NewFramework(DefaultRegistry, handle)
+// inTreeRegistry should contain all known in-tree plugins
+var inTreeRegistry = runtime.Registry{
+	apiresources.APIResourcesName:                           apiresources.NewAPIResources,
+	clusteraffinity.ClusterAffinityName:                     clusteraffinity.NewClusterAffinity,
+	clusterresources.ClusterResourcesFitName:                clusterresources.NewClusterResourcesFit,
+	placement.PlacementFilterName:                           placement.NewPlacementFilter,
+	tainttoleration.TaintTolerationName:                     tainttoleration.NewTaintToleration,
+	clusterresources.ClusterResourcesBalancedAllocationName: clusterresources.NewClusterResourcesBalancedAllocation,
+	clusterresources.ClusterResourcesLeastAllocatedName:     clusterresources.NewClusterResourcesLeastAllocated,
+	clusterresources.ClusterResourcesMostAllocatedName:      clusterresources.NewClusterResourcesMostAllocated,
+	maxcluster.MaxClusterName:                               maxcluster.NewMaxCluster,
+	rsp.ClusterCapacityWeightName:                           rsp.NewClusterCapacityWeight,
```

Comment:
Did we intentionally omit `Fit`?

Reply:
Nope, this is a mistake. Luckily you caught it

Reply:
Wait, it is included already, you might have missed it above